### PR TITLE
Implement option to enable/disable FPS counter for Nvidia and AMD GPU users

### PR DESCRIPTION
**Pull request by Odelpasso**
Hello,

The title explains this commit :D

### CODE REVIEW

**Comment by Odelpasso** [comment]
After reflection, i think that it's not necessary to check if nvidia or amdgpu is loaded to enable the FPS counter. 

FPS env variable are enabled at the same time if the option is enabled in Preference.

**Comment by sharkwouter** [comment]
I'm getting the following error when trying to open the preferences window with this change:
```
Traceback (most recent call last):
  File "/home/wouter/minigalaxy/minigalaxy/window/window.py", line 89, in show_preferences
    preferences_window = Preferences(self, self.config)
  File "/home/wouter/minigalaxy/minigalaxy/window/preferences.py", line 52, in __init__
    self.switch_enable_fps.set_active(self.__config.get("enable_fps"))
TypeError: Argument 1 does not allow None as a value
```

**Comment by Odelpasso** [comment]
Updated boss :D

**Comment by sharkwouter** [comment]
I'm still seeing the same issue :(

**Comment by sharkwouter** [comment]
I've added my commit with the configuration fix. It does work now.

**Comment by sharkwouter** [comment]
I've also tested it on Intel now. With Intel GPUs it doesn't seem to cause any issues.

File: minigalaxy/window/gametile.py [hunk 1]
```diff
@@ -298,6 +298,13 @@ def __start_game(self) -> subprocess:
     def __get_execute_command(self) -> list:
         files = os.listdir(self.__get_install_dir())
 
+        # Enable FPS Counter for Nvidia or AMD (Mesa) users
+        if self.api.config.get("enable_fps") is None:
+            self.apt.config.set("enable_fps", False)
```
Comment (sharkwouter) [inline minigalaxy/window/gametile.py]: Could you rename it to show_fps?

File: data/ui/preferences.ui [hunk 2]
```diff
@@ -165,6 +165,31 @@
...
+                <property name="top_attach">4</property>
+              </packing>
+            </child>
+            <child>
+              <object class="GtkLabel">
+                <property name="visible">True</property>
+                <property name="can_focus">False</property>
+                <property name="halign">start</property>
+                <property name="valign">center</property>
+                <property name="label" translatable="yes">Enable FPS:</property>
```
Comment (sharkwouter) [inline data/ui/preferences.ui]: Could you change it to Show FPS:

**Comment by Odelpasso** [comment]
Boss, there is a small bug and i don't know how to fix that.
How to reproduce : 
- Delete config.json
- Launch Minigalaxy and launch a game --> FPS is not shown/disabled by default
- Go to Preference, enable "Show FPS" and launch a game --> FPS is shown
- Return to Preference, disable "Show FPS" and launch a game --> FPS is still shown and if you check in the config.json, "show fps" is set to False.

**Comment by sharkwouter** [comment]
That bug can be solved by unsetting the environment variables when show fps is disabled in the preferences window.

**Comment by Odelpasso** [comment]
I think is good now :D

**Pull request closed without merging**